packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -166,7 +166,7 @@ function (array $json) use (
                     // decode arrays to objects
                     foreach ($json[Bridge::SYSTEM_ERRORS] as $jsonError) {
                         if (is_string($jsonError)) {
-                            $systemErrors[] = 'System error: ' . $jsonError;
+                            $systemErrors[] = new SystemError('System error: ' . $jsonError);
```

Comment:
Could you type this array, so phpstan would error when strings are pushed onto the array?

Reply:
Add a type to this array, but phpstan does not error when a string is pushed into an array. 🤔

Reply:
It's not supported yet. https://github.com/phpstan/phpstan/issues/3269, https://github.com/phpstan/phpstan/issues/3703